php/boot-fs.php

```diff
@@ -3,7 +3,7 @@
 /**
  * This file needs to parse without error in PHP < 5.3
  */
-if(PHP_SAPI != 'cli') {
+if(PHP_SAPI != 'cli' && PHP_SAPI != 'phpdbg') {
```

Comment:
The only concern I have here is that PHPDBG is for 5.4+, while Terminus supports 5.3+. So long as we don't mind that we would now technically support a module that doesn't work with the full range of PHP versions we want to support, +1